### ISSUE TITLE
ImportInstance selected using SelectModelElement node is reported as UnknownElement

### ISSUE THREAD

**Issue by ThomasMahon** [DynamoDS/DynamoRevit#1410]
## Dynamo version

1.2

## Revit version

Revit 2016

## Operating system

Win 10

## What did you do? 

Selected an ImportInstance (a linked DWG file) using Revit library `SelectModelElement` node

## What did you expect to see?

For the output to identify as a ImportInstance

## What did you see instead?

Dynamo reports an Unknown Element. This causes a number of problems, in particular for Custom Nodes (which triggered my investigation) as any input type declaration that uses ImportInstance breaks the node.

![image](https://cloud.githubusercontent.com/assets/18601851/20670971/5a32e77a-b573-11e6-9fe4-0e984564f00d.png)

**Comment by mjkkirschner** [comment]
@ThomasMahon did this used to work?

**Comment by ThomasMahon** [comment]
Not since I begun testing on V1.0. However, I only discovered the cause of the problem, i.e. UnknownElement, after looking at ways to solve it for a upcoming package release.

**Comment by ThomasMahon** [comment]
@mjkkirschner one other thing to clarify in respect to the error with the CN: the simple solution to prevent the exception is not to declare the type in the Input node. Downside is, there are no hints to the user as to the type of object that needs to be input. Either way, its certainly a problem.

**Comment by ThomasMahon** [comment]
@mjkkirschner also, I've found that if the base class is used (`Revit.Elements.Element`) then no exception is thrown. Only `ImportInstance` class causes this issue.

**Comment by ksobon** [comment]
@ThomasMahon @mjkkirschner this is all happening because for some reason the `ImportInstance` wrapper class was implemented differently than all other wrappers. Mainly, there were initiate methods, fromExisting and Wrapper missing from the class. Once that was added it works as expected: 

![image](https://cloud.githubusercontent.com/assets/529781/21952423/0618461c-d9ea-11e6-8908-46ef53348e48.png)

@mjkkirschner is there a reason not to add these to `ImportInstance` class: 

```
        /// <summary>
        /// Set the internal Element, ElementId, and UniqueId
        /// </summary>
        /// <param name="element"></param>
        private void InternalSetElement(Autodesk.Revit.DB.ImportInstance element)
        {
            InternalImportInstance = element;
            InternalElementId = element.Id;
            InternalUniqueId = element.UniqueId;
        }

        /// <summary>
        /// ImportInstance from existing
        /// </summary>
        /// <param name="element"></param>
        private ImportInstance(Autodesk.Revit.DB.ImportInstance element)
        {
            SafeInit(() => InitElement(element));
        }

        internal static ImportInstance FromExisting(Autodesk.Revit.DB.ImportInstance instance, bool isRevitOwned)
        {
            return new ImportInstance(instance)
            {
                IsRevitOwned = isRevitOwned
            };
        }
```

and this to ElementWrapper: 

```
        public static ImportInstance Wrap(Autodesk.Revit.DB.ImportInstance ele, bool isRevitOwned)
        {
            return ImportInstance.FromExisting(ele, isRevitOwned);
        }
```

**Comment by ksobon** [comment]
@kronz if I am not mistaken this was merged here: https://github.com/DynamoDS/DynamoRevit/pull/1579

**Comment by kronz** [comment]
@aparajit-pratap will this issue be addressed by #1579 ?  Please close if so
